lib/manifest/manifest.js

```diff
@@ -85,36 +84,25 @@ module.exports = function manifestFactory(pluginName, configNameSpace, config) {
   // then add that as the last item in the array
   // then reduce pass them into each other and return as setUp below.
 
-  const molotov = new Molotov();
+  const molotov = Molotov();
```

Comment:
Maybe this var should be lowercase to make it more clear that it's not a class? Also, no config needs to be passed into the factory?

Reply:
Thanks i will put in a follow up patch to add this fix in. This is where we need to pass in cocktailClasses and overrides.

Reply:
Thanks i will put in a follow up patch to add this fix in. This is where we need to pass in cocktailClasses and overrides.